extensions/mound-for-pile/src/utils/types.ts

```diff
@@ -0,0 +1,57 @@
+export interface PilePost {
+  content: string;
+  data: PileDataI;
+  excerpt?: string;
+}
+
+export interface PileDataI {
+  title: string;
+  createdAt: string;
+  updatedAt: string;
+  highlight: string | null;
+  highlightColor: string | null;
+  tags: string[];
+  replies: string[];
+  attachments: string[];
+  isReply: boolean;
+  isAI: boolean;
+}
+
+export interface PileSettings {
+  name: string;
+  path: string;
+  theme: "light" | "dark";
+}
+
+export type NoteI = [string, PileDataI];
+
+export enum PostHightlightsI {
+  None = "None",
+  Highlight = "Highlight",
+  DoLater = "Do later",
+  NewIdea = "New idea",
+}
+
+export const PostHighlights = [
+  {
+    highlight: PostHightlightsI.None,
+    highlightColor: "transparent",
+  },
+  {
+    highlight: PostHightlightsI.Highlight,
+    highlightColor: "#FF703A",
+  },
+  {
+    highlight: PostHightlightsI.DoLater,
+    highlightColor: "#4de64d",
+  },
+  {
+    highlight: PostHightlightsI.NewIdea,
+    highlightColor: "#017AFF",
+  },
+];
```

Comment:
I would rely on `Color` from `@raycast/api` instead, this should always look good if the user is having a theme, is that possible?

Reply:
You can still use the defined colors, its more a heads up that it might go a bit bad with custom themes 🙂

Reply:
got it!. I'll change that up in the next build 😄